ts/features/itwallet/common/utils/itwTypesUtils.ts

```diff
@@ -103,3 +103,5 @@ export type ItwCredentialStatus =
   | "expiring"
   | "expired"
   | ItwJwtCredentialStatus;
+
+export type ItwAuthLevel = undefined | "L2" | "L3";
```

Comment:
Is there a specific use case for `undefined`?

Reply:
Yes, in case the `eid` is not present, we also need to clean up the authentication level used so that mixpanel will track the status ``property`` and ``superProperty`` as not active.

Reply:
The presence of a property should not be tied to its type definition, it introduces opportunities for inconsistencies  in state and type safety.

Let me explain what I mean:

```typescript
export type ItwAuthLevel = "L2" | "L3";
```

`ItwAuthLevel` defines the possible authentication levels we can have, but its absence should not be represented by introducing an additional value within its type.
I think optionality should be explicitly handled at the property level :

```typescript
authLevel?: ItwAuthLevel
```

and/or within parameters:

```typescript
export const itwSetAuthLevel = createStandardAction("ITW_SET_AUTH_LEVEL")<
  ItwAuthLevel | undefined
>();
```

Does this make sense to you?

Reply:
Yes, you are right, this makes sense. Now I don't remember specifically why I had chosen to also use `undefined` in addition to making the parameter optional but on second thought I completely agree with you. Fixed in the last commit, thanks!